### ISSUE TITLE
Integrate lighthouse (Chrome Accessibility Audit)

### ISSUE THREAD

**Issue by chuckhendo** [unumux/pa11y-automated#5]
In addition to using pa11y, we should enable this tool to export results from the Chrome Accessibility Audit. It would be nice to reuse the same puppeteer instance, but I don't even know if that is possible. More research should be done on this
